doc-Appliance_Hardening_Guide/topics/SCAP.adoc

```diff
@@ -1,7 +1,10 @@
 [[_chap_red_hat_cloudforms_security_guide_scap]]
 === Applying SCAP Standards
 
-The Security Content Automation Protocol (SCAP) is a set of standards to assist with vulnerability management and policy compliance. {product-title} provides a set of SCAP standards to apply to your appliance. View these SCAP rules in the `/var/www/miq/lib/appliance_console/config/scap_rules.yml` file.
+The Security Content Automation Protocol (SCAP) is a set of standards to assist
+with vulnerability management and policy compliance. {product-title} provides a
+set of SCAP standards to apply to your appliance. View these SCAP rules in the
+`./var/www/miq/vmdb/productization/appliance_console/config/scap_rules.yml` file.
```

Comment:
dot (.) is not required at the start.
'/var/www/miq/vmdb/productization/appliance_console/config/scap_rules.yml' is seems enough.

Reply:
Thanks! I'll remove it.